internal/sentry/sentry.go

```diff
@@ -0,0 +1,67 @@
+package sentry
+
+import (
+	"log"
+	"os"
+	"task-runner-launcher/internal/logs"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+)
+
+// Configuration options for Sentry. A subset of what is defined in
+// https://docs.sentry.io/platforms/go/configuration/options/
+type Config struct {
+	IsEnabled      bool
+	Dsn            string
+	Release        string
+	Environment    string
+	DeploymentName string
+}
+
+// Init initializes the Sentry client using configuration from the environment.
+// If SENTRY_DSN env var is not set, Sentry will be disabled.
+func Init() {
+	config := ConfigFromEnv()
+	if !config.IsEnabled {
+		return
+	}
+
+	logs.Infof("Initializing Sentry")
```

Comment:
Maybe we should log once it's initialized, else it'll be confusing to show this when the user is not passing in a DSN and so it's disabled.